File: we-marvel-frontend/src/firebaseConfig.js
import {initializeApp} from 'firebase/app';
import {getAuth} from 'firebase/auth';

const firebaseConfig = {
    apiKey: "AIzaSyDlji_5zMWzcvaor8L9msj9isY_oXTOR60",
    authDomain: "wemarvel-f6594.firebaseapp.com",
    projectId: "wemarvel-f6594",
    storageBucket: "wemarvel-f6594.appspot.com",
    messagingSenderId: "1076038087332",
    appId: "1:1076038087332:web:ce21bba1c07bd77b8bac31"
};

export const firebaseApp = initializeApp(firebaseConfig);
export const auth = getAuth(firebaseApp);

File: we-marvel/src/main/java/com/wemarvel/wemarvel/config/FirebaseConfig.java
package com.wemarvel.wemarvel.config;

import com.google.auth.oauth2.GoogleCredentials;
import com.google.firebase.FirebaseApp;
import com.google.firebase.FirebaseOptions;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.autoconfigure.security.SecurityProperties;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Primary;

import java.io.FileInputStream;
import java.io.IOException;
import java.io.InputStream;

@Configuration
public class FirebaseConfig {

    @Autowired
    SecurityProperties secProps;

    @Primary
    @Bean
    public void firebaseInit() {
        InputStream inputStream = null;
        try {
            inputStream = new FileInputStream("C:\\Users\\Imbiamba\\.firebase_config.json");
        } catch (IOException e3) {
            e3.printStackTrace();
        }
        try {

            assert inputStream != null;
            FirebaseOptions options = new FirebaseOptions.Builder()
                    .setCredentials(GoogleCredentials.fromStream(inputStream))
                    .setStorageBucket("wemarvel-f6594.appspot.com")
                    .build();

            if (FirebaseApp.getApps().isEmpty()) {
                FirebaseApp.initializeApp(options);
            }

        } catch (IOException e) {
            e.printStackTrace();
        }
    }

}
